### PR TITLE
[Samples][Java] Fix disparities of 07.using-adaptive-cards sample

### PR DESCRIPTION
**Pull request by VictorGrycuk**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

File: samples/java_springboot/07.using-adaptive-cards/src/main/java/com/microsoft/bot/sample/usingadaptivecards/AdaptiveCardsBot.java [hunk 1]
```diff
@@ -69,7 +69,7 @@
             if (!member.getId().equals(turnContext.getActivity().getRecipient().getId())) {
                  turnContext.sendActivity(
                      String.format("Welcome to Adaptive Cards Bot %s. %s", member.getName(), welcomeText)
-                 ).join();
+                 );
```
Comment (Batta32) [inline samples/java_springboot/07.using-adaptive-cards/src/main/java/com/microsoft/bot/sample/usingadaptivecards/AdaptiveCardsBot.java]: This is not the correct implementation. As mentioned in Slite, you can follow [this pattern](https://github.com/microsoft/BotBuilder-Samples/blob/Java-R13/samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java#L58-L75).

The join should be removed, but the `CompletedFuture` also should be managed.

File: samples/java_springboot/07.using-adaptive-cards/README.md [hunk 2]
```diff
@@ -42,18 +44,62 @@ A message exchange between user and bot can contain media attachments, such as c
...
+- [Spring Boot](https://spring.io/projects/spring-boot)
+- [Maven Plugin for Azure App Service](https://github.com/microsoft/azure-maven-plugins/tree/develop/azure-webapp-maven-plugin)
 - [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
 - [Adaptive Cards](https://adaptivecards.io/)
 - [Send an Adaptive card](https://docs.microsoft.com/en-us/azure/bot-service/nodejs/bot-builder-nodejs-send-rich-cards?view=azure-bot-service-3.0&viewFallbackFrom=azure-bot-service-4.0#send-an-adaptive-card)
 - [Activity processing](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-activity-processing?view=azure-bot-service-4.0)
 - [Azure Bot Service Introduction](https://docs.microsoft.com/azure/bot-service/bot-service-overview-introduction?view=azure-bot-service-4.0)
 - [Azure Bot Service Documentation](https://docs.microsoft.com/azure/bot-service/?view=azure-bot-service-4.0)
-- [.NET Core CLI tools](https://docs.microsoft.com/en-us/dotnet/core/tools/?tabs=netcore2x)
-- [Azure CLI](https://docs.microsoft.com/cli/azure/?view=azure-cli-latest)
```
Comment (Batta32) [inline samples/java_springboot/07.using-adaptive-cards/README.md:57]: Why this was removed?

File: samples/java_springboot/07.using-adaptive-cards/README.md [hunk 3]
```diff
@@ -42,18 +44,62 @@ A message exchange between user and bot can contain media attachments, such as c
...
 - [Azure Portal](https://portal.azure.com)
 - [Channels and Bot Connector Service](https://docs.microsoft.com/en-us/azure/bot-service/bot-concepts?view=azure-bot-service-4.0)
```
Comment (Batta32) [inline samples/java_springboot/07.using-adaptive-cards/README.md:107]: We are adding the `Azure for Java cloud developers` link too (see [here](https://github.com/microsoft/BotBuilder-Samples/blob/Java-R13/samples/java_springboot/03.welcome-user/README.md#further-reading))

**Review by Batta32 — CHANGES_REQUESTED** [review]
Also, in the README:
- `To try this sample locally` -> `To try this sample`

File: samples/java_springboot/07.using-adaptive-cards/README.md (hunk 2 above)
Comment (VictorGrycuk) [reply samples/java_springboot/07.using-adaptive-cards/README.md:57]: This is what you said on Slite that was being removed from all the samples

Comment (VictorGrycuk) [reply samples/java_springboot/07.using-adaptive-cards/README.md:57]: I will revert this change, because I saw that sample 16 also have these links

**Review by Batta32 — APPROVED** [review]

**Pull request closed without merging**